### DISCUSSION TITLE
Direction of project

### DISCUSSION THREAD

**Discussion by BeatLink** [alainm23/planify#716] [General]
Hey @alainm23, I wanted to hear from you where you want to take Planner. You started planner as a native app for todoist but it has grown a lot in scope and shifted in features somewhat. So I was thinking we could discuss the ultimate goal and vision of planner. I think determining this would help to guide a lot of other choices in terms of what features to implement or what areas to focus on. 

For example, here are some paths you could take:

### Keep Planner's focus on being a native Todoist app. 

In this case you would focus on matching the features, styles and design choices of Todoist, including its user interface as well. In this scenario, you would always be looking upstream to todoist for guidance on what to implement or change.

### Expand Planner as an independent, full featured todo-app

In this case, you would focus less on matching the design choices of todoist and instead you would focus on following our own design choices, and implementing the features you and the community would find useful. The app would still be compatible with todoist but it would be only one of the supported platforms. For example, we could work to integrate caldav support, todo.txt support and so on.

**Comment by rzelnik** [comment]
Just my subjective view: Todoist is a proprietary software-as-a-service and therefore any independent native app is a second class citizen for Todoist, only that if would be officially accepted and adopted by Doist Inc. - the Todoist owner. I would prefer a second option - Planner as an independent, full featured todo-app.

**Reply by BeatLink** [reply]
I agree.
